tests/e2e/e2e_test.go

```diff
@@ -205,6 +206,18 @@ func (ctx *testCtx) agentInSync() bool {
 	return true
 }
 
+// execVppctl returns output from vppctl for given action and arguments.
+func (ctx *testCtx) execVppctl(action string, args ...string) (*bytes.Buffer, error) {
```

Comment:
I think it would be more practical to return output as string, it most cases it would be followed by `stdout.String()` anyway.